docs/sources/tempo/traceql/_index.md

```diff
@@ -204,52 +198,100 @@ Find traces where the average duration of the spans in a trace is greater than `
 avg(duration) > 20ms
 ```
 
-For example, find traces that have more than 3 spans with an attribute `http.status` with a value of `200`:
+For example, find traces that have more than 3 spans with an attribute `http.status_code` with a value of `200`:
 
 ```
-{ span.http.status = 200 } | count() > 3
+{ span.http.status_code = 200 } | count() > 3
 ```
 
 ## Arithmetic
 
 TraceQL supports arbitrary arithmetic in your queries. This can be useful to make queries more human readable:
 ```
-{ span.bytesProcessed > 10 * 1024 * 1024 }
+{ span.http.request_content_length > 10 * 1024 * 1024 }
 ```
 to compare the ratios of two span attributes:
 ```
-{ span.bytesProcessed < span.jobsProcessed * 10 }
+{ span.bytes_processed < span.jobs_processed * 10 }
 ```
 or anything else that comes to mind.
 
 ## Examples
 
-Find any trace with a span attribute or resource attribute `namespace` set to `prod`:
+### Simple query to find traces of a specific operation
+
```

Comment:
Adding an introduction to this query. Not sure this write up is phrased correct. Please feel free to modify. 
```suggestion

Let's say that you want to find traces of a specific operation to find a resource with the `frontend` service name and using the name `POST /api/order`. In the example below, both the resource and the name must be present for any results to be returned.  

```